Prototope/TextLayer.swift

```diff
@@ -56,7 +95,31 @@ public class TextLayer: Layer {
 			updateSizePreservingOrigin() // Adjust width/height as necessary for new wrapping mode.
 		}
 	}
-
+	
+	public var textAlignment: Alignment {
+		get {
+			return Alignment(nsTextAlignment: label.textAlignment)
+		}
+		set {
+			label.textAlignment = newValue.toNSTextAlignment()
+			//No need to adjust size, since changing alignment doesn't influence it
+		}
+	}
+	
+	/** Distance from top of layer to the first line's baseline */
+	public var baselineHeight: Double {
+		return Double(label.font.ascender)
+	}
+	
+	/** Aligns this layer with the first baseline of the other layer */
+	public func alignWithBaselineOf(otherLayer: TextLayer) {
+		var delta = pixelAwareCeil(otherLayer.baselineHeight-baselineHeight)
+		
+		var frame = self.frame
+		frame.origin.y = otherLayer.frame.minY + delta
+		self.frame = frame
```

Comment:
In Swift you can actually change the property you want:
self.frame.origin.y = ...